data/npc/the_queen_of_the_banshees.lua

```diff
@@ -59,7 +59,7 @@ local function creatureSayCallback(npc, creature, type, message)
 		return false
 	end
 
-	if MsgContains(message, "seventh seal") then
+	if MsgContains(message, "seventh seal") or MsgContains(message, "seventh") then
```

Comment:
With this pull request we do not need more of this change: https://github.com/opentibiabr/otservbr-global/pull/613